packages/rocketchat-ui-admin/client/adminFlex.html

```diff
@@ -1,13 +1,15 @@
 <template name="adminFlex">
 	<aside class="sidebar-light" role="navigation">
-		<header class="sidebar__header">
-			{{> accountBox lighten=true modifier="--admin"}}
-			<button class="sidebar-flex__back-button" data-action="back">
-				{{> icon block="sidebar-flex__back-icon" icon="back"}}
-				<span>{{_ "Back_to_chat"}}</span>
-			</button>
-		</header>
-		<div class="rooms-list" aria-label="{{_ "Administration"}}">
+		{{#unless embeddedVersion}}
+			<header class="sidebar__header">
+				{{> accountBox lighten=true modifier="--admin"}}
```

Comment:
Also remove this account box

Reply:
I can remove that, but I won't do that on this PR. This PR is about embedded view and not about accountBox.